cmd/main.go

```diff
@@ -131,18 +134,46 @@ func main() {
 		mgr.GetClient(),
 		setupLog,
 	); err != nil {
-		setupLog.Error("Unable to initialize the default reference install template ConfigMaps",
+		setupLog.Error("Unable to initialize the default reference installation template ConfigMaps",
 			zap.Error(err))
 		os.Exit(1)
 	}
 
+	// Create initial SiteConfig Operator configuration
+	operatorConfig, err := getSiteConfigConfiguration(context.TODO(), mgr.GetClient(), siteConfigNamespace, setupLog)
```

Comment:
Why do this here and in the controller?

Reply:
The SiteConfig configuration needs to be initialized (if not present). Note that the ClusterInstance controller needs information about the number of concurrent reconciles during initialization thereof.